### ISSUE TITLE
Add a possibility to clear alerts

### ISSUE THREAD

**Issue by SomeBdyElse** [WeblateOrg/weblate#3227]
**Is your feature request related to a problem? Please describe.**
My translation repo had some merge issues and produced two warnings. As I use the same repo for multiple components, I can see the warning sign on each component. Each component shows me the same two messages. One is 11h old, the other >= 7days. By now, I resolved the underlying issues by merging the git manually. After that, I went to the "Repository maintenance" screen of the main component and made sure I could push, pull and reset. All that was possible, yet, I am still stuck with the warnings. I wonder if there is any way to have weblate recheck and situation and to tell me wether the warnings are still current. 

**Describe the solution you'd like**
Some kind of re-check button or shell command

**Describe alternatives you've considered**
I tried
```
weblate updatechecks --all
weblate updategit --all
weblate lock_translation --all
weblate unlock_translation --all
```
I also tried to somehow dismiss the alerts through the UI

**Comment by nijel** [comment]
Really the alerts should disappear in such case. What Weblate version do you use? This sounds like issue which was fixed by 7c6e1a613b1fb4d3f9e26c978bab4b22949bc331

**Comment by SomeBdyElse** [comment]
I am sorry, you are right. My report was based on an outdated version of weblate. I applied the 7c… change as a patch, now it works like a charm. 
Thank you!
